src/alire/alire-crate_configuration.adb

```diff
@@ -151,6 +151,9 @@ package body Alire.Crate_Configuration is
         (File, "--  Configuration for " & (+Crate) & " generated by Alire");
 
       TIO.Put_Line (File, "package " & (+Crate) & "_Config is");
+      TIO.Put_Line (File, "   pragma Pure;");
+      TIO.Put_Line (File, "   pragma No_Elaboration_Code_All;");
+      TIO.Put_Line (File, "   pragma Preelaborate;");
```

Comment:
I think that `Preelaborate` is implicit in `Pure`, so removing the former might save a warning down the line.

Reply:
Indeed :+1: